File: Python/Pattern/07.startriangle.py
'''Create a pattern like this

       *
     ***
   *****
  *******
*********

'''

row = int(input("Enter the no. of rows: "))

for i in range (row):
    for j in range (row - i -1):
        print(" ", end='')
        
    for k in range(2 * i +1):
        print("*", end='')
        
    print("\n")

File: CPP/Patterns/07.startriangle.cpp
/*Create a pattern like this

       *
     ***
   *****
  *******
*********

*/

#include <iostream>
using namespace std;

int main() {
    int row;
    cout<<"Enter no. of rows: ";
    cin>>row;
    
    for (int i =0; i< row; i++){
        for (int j =0; j < row-i -1; j++){
            cout<<" ";
        }
        for (int k = 0; k < 2*i+1; k++){
            cout<<"*";
        }
        cout<<endl;
    }

    return 0;
}